packages/neo4j-driver/test/rx/session.test.js

```diff
@@ -442,6 +442,111 @@ describe('#unit rx-session', () => {
     })
   })
 
+  describe('.run()', () => {
+    it('should redirect run to the underline session', () => {
+      const capture = []
+      const _session = {
+        run: (...args) => {
+          capture.push(args)
+          const summary = new ResultSummary(args[0], args[1], {}, 5.3)
+          return {
+            async * [Symbol.asyncIterator] () {
+              return summary
+            },
+            async summary () {
+              return summary
+            }
+          }
+        }
+      }
+      const session = new RxSession({ session: _session })
+      const query = 'the query'
+      const params = {}
+      const txConfig = { timeout: 1000 }
```

Comment:
Does it make sense to use a non-int timeout here to verify it gets passed down unchanged?

Reply:
I just want to check if the value is send to `run` since I known Session.run knows how to handle this value

Reply:
Fair enough. I was just thinking it might be good to test that RxSession.run does not change the timeout even if it has sub-ms information. But it's no big deal either way.